src/convertToObject.js

```diff
@@ -3,16 +3,33 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
+    .split(';')
+    .reduce((acc, words) => {
+      const [key, value] = words
+        .split(':')
+        .map((word) => word.trim());
+
+      if (key === '') {
+        return acc;
+      }
```

Comment:
```suggestion
      if (!key) {
        return acc;
      }
```
You don't need to strictly compare with an empty string, just check if a value exists
https://developer.mozilla.org/en-US/docs/Glossary/falsy